ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -197,10 +199,40 @@ const getVariantLocus = (variant, project) => {
   )
 }
 
-const IgvPanel = React.memo((
-  { variant, igvSampleIndividuals, individualsByGuid, project, sampleTypes, rnaReferences, minJunctionEndsVisible },
+const getGeneLocus = (variant, genesById, project) => {
+  if (variant.transcripts) {
+    const geneIds = Object.keys(variant.transcripts)
+    const geneId = geneIds.length > 0 ? geneIds[0] : ''
```

Comment:
Which gene will be displayed in the case of multiple genes for a variant?

Reply:
the main gene, there is a `getVariantMainGeneId` helper function you can use